src/arrayMethodSort.js

```diff
@@ -3,9 +3,35 @@
 /**
  * Implement method Sort
  */
+const compareAsStrings = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
+
+  if (stringA === stringB) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareAs = compareAsStrings) {
```

Comment:
While the bubble sort algorithm is correctly implemented, it's worth noting that bubble sort has a worst-case and average complexity of O(n^2), which is not efficient for large datasets. If performance is a concern, consider implementing a more efficient sorting algorithm like quicksort or mergesort.